src/WinRT.Runtime/Projections/INotifyCollectionChanged.net5.cs

```diff
@@ -45,11 +46,21 @@ public static unsafe (Action<global::System.Collections.Specialized.NotifyCollec
     [DynamicInterfaceCastableImplementation]
     [EditorBrowsable(EditorBrowsableState.Never)]
     [Guid("530155E1-28A5-5693-87CE-30724D95A06D")]
+    [WuxMuxProjectedType]
     internal unsafe interface INotifyCollectionChanged : global::System.Collections.Specialized.INotifyCollectionChanged
     {
-        [Guid("530155E1-28A5-5693-87CE-30724D95A06D")]
+        public static string GetGuidSignature()
+            => Projections.UiXamlModeSetting == Projections.UiXamlMode.WindowsUiXaml
+                ? "{CF75D69C-F2F4-486B-B302-BB4C09BAEBFA}"
+                : typeof(INotifyCollectionChanged).GUID.ToString("B");
```

Comment:
Do we want to hardcode the GUID string in both cases (for consistency and marginally better perf)?

Reply:
I can go back and do that.